src/ToDo.UI/Strings/en/Resources.resw

```diff
@@ -1,192 +1,310 @@
+<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--This file is generated from 'Resources.l12n.xml'. Do not modify it manually.-->
-  <data name="ApplicationName" xml:space="preserve">
+	<!-- 
```

Comment:
@Xiaoy312 with your script for creating the resources files, can you please add this header xml to make sure the resources files open correctly in VS